98_exercises.Rmd

```diff
@@ -435,11 +435,14 @@ customise the correlation method and significance testing used for the analysis.
 
 ### Multi-omic Factor Analysis (MOFA)
 
-We further move onto an exercise on MOFA.
+TO DO
 
-* Move further onto multi-omics factor analysis (\@ref(mofa)).
+We further move onto an exercise on MOFA, which is explained in chapter \@ref(mofa).
```

Comment:
Does this mean that MOFA exercise is in the MOFA chapter and not in the exercise chapter?

Wouldn't it be beneficial to have all exercises in one place (exercise chapter), and then refer to those in the text? Or alternatively have all exercises embedded in the text chapters (which I think will be more difficult to maintain and develop further).

Reply:
No sorry. It just means that I still have to make an exercise about it (if necessary). Otherwise I'll just delete this part

Reply:
This is ok.